docs/modules/language-tutorial/pages/02_filling_out_a_template.adoc

```diff
@@ -206,6 +206,8 @@ Going back to the example above, you can write `parrot` as a separate module.
 name = "Common wood pigeon"
 diet = "Seeds"
 taxonomy {
+  kingdom = "Animalia"
+  clade = "Dinosauria"
```

Comment:
IANB (I Am No Biologist; let's make that a thing), but...
```suggestion
  clade = "Columbimorphae"
```

Reply:
👍